src/web/middleware/sentry.rs

```diff
@@ -52,6 +55,28 @@ pub struct SentryWrapperMiddleware<S> {
     service: Rc<RefCell<S>>,
 }
 
+impl<S, B> SentryWrapperMiddleware<S>
+where
+    S: Service<Request = ServiceRequest, Response = ServiceResponse<B>, Error = Error> + 'static,
+    S::Future: 'static,
+    B: 'static,
+{
+    fn clean_uri(&self, uri: &Uri) -> String {
+        // Remove the variable userID and batch tokens from the URI tags
+        lazy_static! {
+            // most values are either "=true" or a lower case hex string.
+            static ref RE: regex::Regex = regex::Regex::new(r"=[a-z\d]{5,}").unwrap();
```

Comment:
Just to be sure I understand, this matches anything starting with = followed with any lowercase letter or number; and it has to match at least 5 letters or numbers in order to not have it match on the string `=true`. Is that right?

Reply:
Yep, that's it exactly. I was debating about making it match case independent, but most of the ones I'm seeing are lower case.  If folks feel otherwise, it's an easy fix.